kubernetes/secrets/synapse-config/synapse.log.config.erb

```diff
@@ -0,0 +1,28 @@
+version: 1
```

Comment:
this doesn't seem to have any secrets... maybe we should build this into the container lol

Reply:
any suggestions for where I should mount this?